Modules/System/Cues and KPIs/src/CuesAndKPIsImpl.Codeunit.al

```diff
@@ -121,7 +121,12 @@ codeunit 9702 "Cues And KPIs Impl."
     local procedure GetCustomizedCueStyleOption(TableId: Integer; FieldNo: Integer; CueValue: Decimal): Integer
     var
         CueSetup: Record "Cue Setup";
+        CueStyle: Enum "Cues And KPIs Style";
+        Resolved: Boolean;
     begin
+        CuesAndKPIs.OnBeforeGetCustomizedCueStyleOption(TableID, FieldNo, CueValue, CueStyle, Resolved);
+        if Resolved then
+            exit(Style);
```

Comment:
`CueStyle` maybe? 🤔

Reply:
Indeed. I was too quick with the changes. Corrected.